### PR TITLE
Dockerfiles

### PR DESCRIPTION
**Pull request by aknguyen7**
docker files for CE

### CODE REVIEW

**Comment by aknguyen7** [comment]
relate to https://github.com/OpenLiberty/openliberty.io/issues/2595

File: docker/Dockerfile.builder [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#
+#
+# One major factor for using Ubuntu 18.04 LTS is to stay on an older Python 3.6 until the python
+# scripts are migrated to use a higher version
```
Comment (kinueng) [inline docker/Dockerfile.builder]:
```suggestion
# One major factor for using `pipeline-base-image:2.6` is to stay on an older Python 3.6 until the python
# scripts are migrated to use a higher version
```

File: docker/Dockerfile.builder [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+FROM icr.io/continuous-delivery/pipeline/pipeline-base-image:2.6 as builder
+SHELL ["/bin/bash", "-c"]
+
+COPY scripts /scripts
+
+RUN apt-get update && \
+    apt-get -y install curl gnupg build-essential ruby-full git python3 python3-bs4 python3-lxml \
+    nodejs libgdbm-dev libncurses5-dev automake libtool bison libffi-dev
```
Comment (kinueng) [inline docker/Dockerfile.builder]: I see `nodejs` is part of this list. What is the difference between doing `apt-get install nodejs` vs `node_install.sh`?

File: scripts/build/node_install.sh [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
+echo "$PATH"
+echo "node --version"
+node --version
+echo "npm --version"
+npm --version
+echo "ls -g --depth=0"
+npm ls -g --depth=0
 
 timer_end=$(date +%s)
 echo "Total execution time for installing Node: '$(date -u --date @$(( $timer_end - $timer_start )) +%H:%M:%S)'"
```
Comment (kinueng) [inline scripts/build/node_install.sh]:
```suggestion
echo "Total execution time for installing Node: '$(date -u --date @$(( $timer_end - $timer_start )) +%H:%M:%S)'"

```

Adding EOL to file

File: scripts/build/node_install.sh [hunk 4]
```diff
@@ -1,14 +1,23 @@
...
 echo "Install Node"
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
-export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
-nvm install --lts
-
-echo "npm version:"
-echo `npm -v`
+#comment this below line in local if nvm already installed
+export NVM_DIR=/root/.nvm
```
Comment (kinueng) [inline scripts/build/node_install.sh:7]: This seems like something specific to only the Docker build environment. Could we move `export NVM_DIR=/root/.nvm` to the Dockerfile?

Comment (aknguyen7) [reply scripts/build/node_install.sh:7]: we are using `NVM_DIR` in line 9-12

File: docker/Dockerfile.builder (hunk 2 above)
Comment (aknguyen7) [reply docker/Dockerfile.builder]: i think they are the same - to install node, will remove `nodejs` part of this list and test it out in demo2

**Review by kinueng — APPROVED** [review]

**Pull request merged**